pyproject.toml

```diff
@@ -64,32 +64,32 @@ optional-dependencies."tests" = [
 urls."bug tracker" = "https://github.com/gammasim/simtools/issues"
 urls."documentation" = "https://gammasim.github.io/simtools/"
 urls."repository" = "https://github.com/gammasim/simtools"
-scripts.simtools-add-file-to-db = "simtools.applications.add_file_to_db:main"
-scripts.simtools-add-value-from-json-to-db = "simtools.applications.add_value_from_json_to_db:main"
 scripts.simtools-calculate-trigger-rate = "simtools.applications.calculate_trigger_rate:main"
-scripts.simtools-compare-cumulative-psf = "simtools.applications.compare_cumulative_psf:main"
 scripts.simtools-convert-all-model-parameters-from-simtel = "simtools.applications.convert_all_model_parameters_from_simtel:main"
 scripts.simtools-convert-model-parameter-from-simtel = "simtools.applications.convert_model_parameter_from_simtel:main"
+scripts.simtools-db-add-file-to-db = "simtools.applications.db_add_file_to_db:main"
 scripts.simtools-db-add-model-parameters-from-repository-to-db = "simtools.applications.db_add_model_parameters_from_repository_to_db:main"
+scripts.simtools-db-add-value-from-json-to-db = "simtools.applications.db_add_value_from_json_to_db:main"
 scripts.simtools-db-get-array-layouts-from-db = "simtools.applications.db_get_array_layouts_from_db:main"
+scripts.simtools-db-get-file-from-db = "simtools.applications.db_get_file_from_db:main"
+scripts.simtools-db-get-parameter-from-db = "simtools.applications.db_get_parameter_from_db:main"
 scripts.simtools-derive-mirror-rnda = "simtools.applications.derive_mirror_rnda:main"
```

Comment:
derive_mirror_rnda and tune_psf is quite similar. Maybe both derive? Or we group everything (derivations, tuning) under `calculate` (with the trigger rates)?

Reply:
Very good suggestion: renamed tune_psf to derive_psf_parameters

Note the difference between deriving model parameters (which is different to calculating e.g., trigger rates), so I think having `calculate...` and `derive...` is consistent.

Reply:
Ok this sounds good. You can merge.